lib/finitomata.ex

```diff
@@ -270,6 +287,9 @@ defmodule Finitomata do
     @typedoc "The payload that has been passed to the FSM instance on startup"
     @type payload :: any()
 
+    @typedoc "The parent process for this particular FSM implementation"
```

Comment:
I would add that nil means, there is not parent, and this process is at the top of the hierarchy. That is If I understood correctly.

Reply:
“Parent” is a bad name, but I was unable to come out with a better one. This is basically the process that called `Finitomata.start_fsm/4`, and it always exists.